src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/VersionTests.kt

```diff
@@ -123,8 +120,8 @@ class VersionTests : CleanDatabaseTests()
         Assertions.assertThat(results[0].latestVersion).isEqualTo("vz")
         Assertions.assertThat(results[0].id).isEqualTo("va")
         Assertions.assertThat(results[0].published).isTrue()
-        Assertions.assertThat(results[0].author).isEqualTo("author authorson")
-        Assertions.assertThat(results[0].requester).isEqualTo("requester mcfunder")
+        Assertions.assertThat(results[0].customFields["author"]).isEqualTo("author authorson")
```

Comment:
Might be good to also have a test where not all records have all custom fields, to test that the keys are still present as desired.